app/assets/images/payment_icons/payflex.svg

```diff
@@ -1 +1 @@
7 1.07 1.07 0 0 0 .28-.19l.2-.21a5.62 5.62 0 0 0 .81-1.22A4.87 4.87 0 0 0 3.78 13a4.87 4.87 0 0 0-1.22.41 5.62 5.62 0 0 0-1.22.8zM14.54 11.35a1.85 1.85 0 0 0-.25-.53 1.41 1.41 0 0 0-.43-.37 1.31 1.31 0 0 0-.61-.13 1.33 1.33 0 0 0-.63.14 1.4 1.4 0 0 0-.42.38 1.63 1.63 0 0 0-.23.54 2.49 2.49 0 0 0-.08.61 2.72 2.72 0 0 0 .08.63 1.37 1.37 0 0 0 .25.53 1.22 1.22 0 0 0 .43.38 1.38 1.38 0 0 0 .64.14 1.22 1.22 0 0 0 .63-.15 1.24 1.24 0 0 0 .41-.38 1.92 1.92 0 0 0 .22-.55 2.73 2.73 0 0 0 .07-.64 2.47 2.47 0 0 0-.08-.6m-2.62-1.57v.6a1.09 1.09 0 0 1 .56-.54 2.15 2.15 0 0 1 .85-.17 2.13 2.13 0 0 1 .9.19 2 2 0 0 1 .63.51 2.36 2.36 0 0 1 .38.74 2.9 2.9 0 0 1 .13.89 3.26 3.26 0 0 1-.12.9 2.2 2.2 0 0 1-.38.73 1.74 1.74 0 0 1-.64.49 2 2 0 0 1-.88.18h-.37a2.25 2.25 0 0 1-.4-.12 1.53 1.53 0 0 1-.37-.21 1 1 0 0 1-.29-.33v2.28h-.73V9.78zM18.82 11.92a1 1 0 0 1-.36.15l-.47.07-.49.07a1.84 1.84 0 0 0-.45.13.7.7 0 0 0-.32.24.69.69 0 0 0-.12.43.56.56 0 0 0 .07.3.52.52 0 0 0 .19.2.67.67 0 0 0 .27.11 1.12 1.12 0 0 0 .32 0 1.65 1.65 0 0 0 .62-.1 1.42 1.42 0 0 0 .42-.25.9.9 0 0 0 .24-.32.88.88 0 0 0 .08-.33zM20 14.19a1.09 1.09 0 0 1-.52.11.61.61 0 0 1-.45-.16.65.65 0 0 1-.17-.52 1.67 1.67 0 0 1-.7.52 2.23 2.23 0 0 1-.86.16 2.12 2.12 0 0 1-.57-.07 1.35 1.35 0 0 1-.47-.21 1.15 1.15 0 0 1-.31-.38 1.26 1.26 0 0 1-.12-.57 1.39 1.39 0 0 1 .13-.62 1.17 1.17 0 0 1 .34-.39 1.53 1.53 0 0 1 .48-.23l.55-.13.57-.09a3 3 0 0 0 .48-.08.8.8 0 0 0 .32-.16.42.42 0 0 0 .12-.32.69.69 0 0 0-.09-.38.54.54 0 0 0-.23-.22.82.82 0 0 0-.31-.11h-.35a1.6 1.6 0 0 0-.77.17.75.75 0 0 0-.34.67H16a1.48 1.48 0 0 1 .17-.7 1.3 1.3 0 0 1 .41-.46 1.84 1.84 0 0 1 .59-.25 3.06 3.06 0 0 1 .7-.08 4 4 0 0 1 .6.05 1.57 1.57 0 0 1 .53.17 1.06 1.06 0 0 1 .39.37 1.18 1.18 0 0 1 .14.63v2.27a1.74 1.74 0 0 0 0 .38q0 .12.21.12a.69.69 0 0 0 .22 0zM22.24 14.8a4.88 4.88 0 0 1-.25.55 1.47 1.47 0 0 1-.27.36 1.2 1.2 0 0 1-.34.21A1.33 1.33 0 0 1 21 16h-.26a1 1 0 0 1-.25-.06v-.67l.22.08h.21a.62.62 0 0 0 .37-.11.64.64 0 0 0 .23-.3l.3-.75L20 9.78h.83l1.29 3.62 1.28-3.62h.77zM24.34 10.42v-.64h.75v-.66a1 1 0 0 1 .31-.82 1.31 1.31 0 0 1 .91-.3h.24l.23.05v.63a.65.65 0 0 0-.2 0h-.21a.71.71 0 0 0-.41.11c-.1.06-.14.2-.14.39v.62h.85v.64h-.85v3.79h-.73v-3.81zM27.37 8.09h.73v6.12h-.73zM32 11.08a1.11 1.11 0 0 0-.26-.4 1.2 1.2 0 0 0-.4-.26 1.15 1.15 0 0 0-.5-.1 1.19 1.19 0 0 0-.51.1 1.05 1.05 0 0 0-.39.27 1.14 1.14 0 0 0-.25.39 1.6 1.6 0 0 0-.12.49h2.55a1.36 1.36 0 0 0-.12-.49m.18 2.84a2 2 0 0 1-1.25.38 2.32 2.32 0 0 1-.92-.17 2 2 0 0 1-.66-.48 2 2 0 0 1-.4-.74 3.35 3.35 0 0 1-.15-.93 2.52 2.52 0 0 1 .16-.93 2 2 0 0 1 .43-.72 2 2 0 0 1 1.5-.66 1.85 1.85 0 0 1 1 .25 1.93 1.93 0 0 1 .62.62 2.36 2.36 0 0 1 .33.82 3.61 3.61 0 0 1 .07.85h-3.33a2 2 0 0 0 .07.55 1.33 1.33 0 0 0 .25.47 1.32 1.32 0 0 0 .44.32 1.49 1.49 0 0 0 .62.12 1.22 1.22 0 0 0 .76-.22 1 1 0 0 0 .39-.65h.72a1.77 1.77 0 0 1-.64 1.12" fill="#002751"/><path fill="#002751" d="M32.96 9.78h.93l1.07 1.55 1.1-1.55h.88l-1.53 2.04 1.71 2.39h-.93l-1.23-1.84-1.24 1.84h-.88l1.66-2.33-1.54-2.1z"/></svg>
```

Comment:
It's missing a few elements:

- `aria-labelledby="pi-payflex"` attribute (like the previous icon)
- `<title id="pi-payflex">Payflex</title>` element (like the previous account)
- `width="38" height="24"` attribute (like the previous account)